micropip/_commands/mock_package.py

```diff
@@ -81,85 +80,99 @@ def init_fn(module):
         persist between runs of python (assuming the file system persists).
         If it is False, modules will be stored inside micropip in memory only.
     """
+
     if modules is None:
         # make a single mock module with this name
         modules = {name: ""}
+
     # make the metadata
     METADATA = f"""Metadata-Version: 1.1
 Name: {name}
 Version: {version}
 Summary: {name} mock package generated by micropip
 Author-email: {name}@micro.pip.non-working-fake-host
 """
-    for n in modules.keys():
-        METADATA += f"Provides:{n}\n"
+
+    for module_name in modules.keys():
+        METADATA += f"Provides: {module_name}\n"
 
     if persistent:
         # make empty mock modules with the requested names in user site packages
-        site_packages = Path(site.getusersitepackages())
-        # in pyodide site packages isn't on sys.path initially
-        if not site_packages.exists():
-            site_packages.mkdir(parents=True, exist_ok=True)
-        if site_packages not in sys.path:
-            sys.path.append(str(site_packages))
-        metadata_dir = site_packages / (name + "-" + version + ".dist-info")
-        metadata_dir.mkdir(parents=True, exist_ok=False)
-        metadata_file = metadata_dir / "METADATA"
-        record_file = metadata_dir / "RECORD"
-        installer_file = metadata_dir / "INSTALLER"
-        file_list = []
-        file_list.append(metadata_file)
-        file_list.append(installer_file)
-        with open(metadata_file, "w") as mf:
-            mf.write(METADATA)
-        for n, s in modules.items():
-            if s is None:
-                s = ""
-            s = dedent(s)
-            path_parts = n.split(".")
+        site_packages = Path(site.getsitepackages()[0])
```

Comment:
I am not a fan of this very long conditional, especially since the else is very short and it's basically the end of the function. Ideally we should change this to:
```
   if not persistent:
      # short not persistent code
      return
   # longer persistent code not indented
```
or it could even be factored into a separate function.

I understand this isn't really your fault though @ryanking13. If you want we can clean it up later.

Reply:
Yeah, I'll do some cleanup later.